### PR TITLE
DebugID

### PR DESCRIPTION
**Pull request by eggrobin**

### CODE REVIEW

File: ksp_plugin/part.cpp [hunk 1]
```diff
@@ -144,6 +153,15 @@ not_null<std::unique_ptr<Part>> Part::ReadFromMessage(
   return part;
 }
 
+std::string Part::DebugID() const {
```
Comment (pleroy) [inline ksp_plugin/part.cpp]: Upon reflection I'd call this `ShortDebugString`.

File: ksp_plugin/part.hpp [hunk 2]
```diff
@@ -100,8 +101,12 @@ class Part final {
       serialization::Part const& message,
       std::function<void()> deletion_callback);
 
+  // returns "part name (part ID)".
```
Comment (pleroy) [inline ksp_plugin/part.hpp]: Returns

File: ksp_plugin/vessel.cpp [hunk 3]
```diff
@@ -281,6 +289,10 @@ not_null<std::unique_ptr<Vessel>> Vessel::ReadFromMessage(
   return std::move(vessel);
 }
 
+std::string Vessel::DebugID() const {
```
Comment (pleroy) [inline ksp_plugin/vessel.cpp]: Same.

File: serialization/journal.proto [hunk 4]
```diff
@@ -748,8 +748,9 @@ message InsertOrKeepVessel {
   message In {
     required fixed64 plugin = 1 [(pointer_to) = "Plugin", (is_subject) = true];
     required string vessel_guid = 2;
-    required int32 parent_index = 3;
-    required bool loaded = 4;
+    required string vessel_name = 3;
```
Comment (pleroy) [inline serialization/journal.proto]: Don't renumber the fields, it's a bad practice, just use 5 for the `vessel_name`.

File: serialization/journal.proto [hunk 5]
```diff
@@ -765,11 +766,12 @@ message InsertOrKeepLoadedPart {
   message In {
     required fixed64 plugin = 1 [(pointer_to) = "Plugin", (is_subject) = true];
     required fixed32 part_id = 2;
-    required double mass_in_tonnes = 3;
-    required string vessel_guid = 4;
-    required int32 main_body_index = 5;
-    required QP main_body_world_degrees_of_freedom = 6;
-    required QP part_world_degrees_of_freedom = 7;
+    required string name = 3;
```
Comment (pleroy) [inline serialization/journal.proto]: Same.

File: serialization/journal.proto [hunk 6]
```diff
@@ -781,8 +783,9 @@ message InsertUnloadedPart {
   message In {
     required fixed64 plugin = 1 [(pointer_to) = "Plugin", (is_subject) = true];
     required fixed32 part_id = 2;
-    required string vessel_guid = 3;
-    required QP from_parent = 4;
+    required string name = 3;
```
Comment (pleroy) [inline serialization/journal.proto]: And here too.

File: ksp_plugin/part.cpp (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin/part.cpp]: Done.

**Pull request merged**